swig/python/gdal-utils/README.rst

```diff
@@ -1,23 +1,21 @@
 gdal-utils
 =============
 
-This gdal-utils Python package includes modules and functions that use and extend the
-GDAL_ Geospatial Data Abstraction Library.
+This gdal-utils Python package includes modules and functions that use and
+extend the GDAL_ Geospatial Data Abstraction Library.
 
-The GDAL project (primarily Even Rouault) maintains SWIG generated Python
-bindings for GDAL and OGR, which are included in the gdal package.
+The GDAL project maintains SWIG generated Python bindings for GDAL and OGR,
+which are included in the gdal package.
 
 Please also refer to the osgeo documentation and to the `GDAL API Tutorial`_.
 
 Currently, The the gdal package also includes the utils package,
 but in the future they might be available only in this gdal-utils package.
 
-Because we don't test the utils against a different versions of gdal -
-If you want to get the latest version of the utils, we do recommend you to upgrade
-to the latest version of gdal and get the latest version of both.
+Versioning of GDAL and gdal-utils are independent from each other, e.g.
```

Comment:
I would keep the original text: there's no way we can guarantee that a version of gdal-utils more recent than libgdal or its python bindings will work (as this is not tested and given that "what is not tested is broken" is a good mantra). In most cases, it might seem to work, but for some options, gdal-utils can use newer API or depend on bug fixes.

Reply:
Thanks for that feedback. I re-wrote this paragraph to accommodate and added a new section on how to test compability across versions. It's soft spot is we assume Python Osgeo module is already installed and working. 

(I'm hazy on how this works in Ubuntu, on the relationship between `python3-gdal` apt-get package and PyPi.org `gdal` package. Somehow both seem to be necessary but the latter overrides part of the former, and requires it? Or something.)

Reply:
> (I'm hazy on how this works in Ubuntu, on the relationship between python3-gdal apt-get package and PyPi.org gdal package. Somehow both seem to be necessary but the latter overrides part of the former, and requires it? Or something.)

No, you can't use either. If both are used, generally the python import paths rules will use the pip module.

Reply:
> No, you can't use either. If both are used, generally the python import paths rules will use the pip module.

Sorry I'm having trouble parsing this. Can you expand?  Or maybe my fuzziness on apt vs pip packages doesn't need to be resolved right now or for this PR and can wait. Is the Readme in current state okay?

Reply:
sorry I meant that you can use either the python3-gdal apt-get package or the pypi pip package (but for that later one, you need to make sure to use the same version as the native libgdal installed on the system).

Reply:
> Is the Readme in current state okay?

yes